### PR TITLE
Feature/blog list

### PR DESCRIPTION
**Pull request by nistormihai**

### CODE REVIEW

File: server/liveblog/blogslist/blogslist_assets/main.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    function BlogsList($resource, config) {
+        return $resource(config.api_host + 'api/client_blogs',{}, {
+            'get': {
+                method:'GET',
+                transformResponse: function(blogs) {
+                    blogs = angular.fromJson(blogs);
+                    // Transformation for blog image renditions.
+                    // angular.forEach(function(blog) {
+                    //     blog = transformBlog(blog);
+                    // })
```
Comment (vladnicoara) [inline server/liveblog/blogslist/blogslist_assets/main.js]: will this code be needed? if not please remove commented code

File: server/liveblog/blogslist/blogslist_assets/main.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        .constant('config', {
+            api_host:  window.LB.api_host,
+            assets_root: window.LB.assets_root
+        }).controller('BlogListCtrl',BlogListCtrl)
+        .config(['$routeProvider', 'config', '$sceDelegateProvider',
+              function($routeProvider, config, $sceDelegateProvider) {
+                    $sceDelegateProvider.resourceUrlWhitelist(['https://*.s3-eu-west-1.amazonaws.com/**'])
+                    $routeProvider.
+                      when('/', {
+                              'template': "<div ng-repeat=\"blog in bl.blogs._items track by blog._id\" ng-click=\"bl.openBlog(blog)\" class=\"content-item\"><h3>{{ ::blog.title }}</h3><div class=\"list-item\"><div class=\"list-item__author\"><span>by</span> {{ ::blog.original_creator | username }}</div><div class=\"list-item__time\"><span>{{ ::\'Created\' | translate}}</span> {{ ::blog._created | reldate }} <span class=\"list-item--margin\">{{ ::\'updated\' | translate }}</span> {{ ::blog._updated | reldate }}.</div><div class=\"list-item__description\" ng-if=\"blog.description\">{{ ::blog.description | htmlToPlaintext }}</div></div></div>",
```
Comment (vladnicoara) [inline server/liveblog/blogslist/blogslist_assets/main.js]: maybe it's best for this template to be in a file as it is a bit large

File: CUSTOM-THEME.MD [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+                `type`: checkbox, number, select, datetimeFormat
+                `default`: the initial value
+                `help`: visual to be clear what it does in the UI.
+                `options`: a list of objects with `value` and `label` properties for `select type` 
+
+* README.md a md text with information regarding the theme.
+
+a theme may have `screenshot.png` file for a theme preview in the UI.
+
+we recommend that a used of the gulp file from the classic or angular theme is used.
```
Comment (vladnicoara) [inline CUSTOM-THEME.MD]: maybe you can rephrase and be a bit clearer here

File: server/liveblog/blogslist/blogslist_assets/main.js (hunk 2 above)
Comment (nistormihai) [inline server/liveblog/blogslist/blogslist_assets/main.js]: because s3, having in a different file will pose greater effort

**Pull request merged**